extensions/stackdriver/log/logger.cc

```diff
@@ -177,6 +178,10 @@ bool fillAuthInfo(const std::string& response_details,
     (*label_map)["policy_rule"] = policy_rule_index;
     return true;
   }
+  if (RE2::PartialMatch(response_details, rbac_denied_match_default)) {
```

Comment:
Can we refactor the code a little bit:

1. Check if it has the prefix "rbac_denied_match_default"
2. If it has the prefix, set response_details to "AuthzDenied", otherwise just skip
3. Parse the remaining part for the ns, name and rule. Skip if the parse failed

So that you only need 1 regex match, set the response_details once and only need 1 definition of "rbac_denied_match_default".

Reply:
@yangminzhu : what is the scenario, we will always have? If it's going to be rbac denied match with full policy name and rule, then we should keep it as is, as it gets away with just 1 regex match. But if the second scenario of none policy match is more common, then we should do what you suggested...

Reply:
The second scenario (none policy matched) will be more common because ALLOW action is used much more widely.

Reply:
Cool.. fixed